### ISSUE TITLE
NX Cannot read properties of undefined (reading 'endsWith')

### ISSUE THREAD

**Issue by makhataibar** [nrwl/nx-labs#98]
After I tried update to last version (14.5) nx and @nrwl/expo, I have errors on CI when `yarn install`:
<img width="1112" alt="Screen Shot 2022-08-04 at 08 33 01" src="https://user-images.githubusercontent.com/27006656/182750611-d77b8871-4efb-45e0-be5b-ca355657a629.png">

**Comment by xiongemi** [comment]
@makhataibar what version of `nx` are you using?
could you run `nx report` in your repo and paste your response here?

**Comment by xiongemi** [comment]
@makhataibar I think it might be a duplicate of https://github.com/nrwl/nx-labs/issues/87.

if the migration script did not apply, you might manually update in your app's metro.config.js, in `return withNxMetro...`, add 
```
   projectRoot: __dirname,
    watchFolders: [],
```

so it becomes:
```
return withNxMetro(defaultConfig, {
    // Change this to true to see debugging info.
    // Useful if you have issues resolving modules
    debug: false,
    // all the file extensions used for imports other than 'ts', 'tsx', 'js', 'jsx'
    extensions: [],
    projectRoot: __dirname,
    watchFolders: [],
  });
```

**Comment by makhataibar** [comment]
@xiongemi I changed metro.config.js, but still throw error 
<img width="1728" alt="Screen Shot 2022-08-12 at 11 05 44" src="https://user-images.githubusercontent.com/27006656/184288442-fb7a5ef6-8d69-4d46-b00f-36d86964d297.png">

**Comment by makhataibar** [comment]
it was problem with yarn.lock . I don't why, but after upgrade to nx@14.5 nx version was fixed to 14.0.1 for @nrwl/expo.
I just remove node_modules and yarn.lock and reinstall
